packages/android_intent_plus/example/integration_test/android_intent_plus_e2e.dart

```diff
@@ -1,18 +1,19 @@
 import 'dart:io';
```

Comment:
Please rename the file to `android_intent_plus_test.dart`

Reply:
I think I have finished it :D